src/modules/mavlink/mavlink_params.c

```diff
@@ -141,3 +141,27 @@ PARAM_DEFINE_INT32(MAV_FWDEXTSP, 1);
  * @group MAVLink
  */
 PARAM_DEFINE_INT32(MAV_BROADCAST, 0);
+
+/**
+ * Disable parameter hash check.
+ *
+ * Disabling the parameter hash check functionality will make the mavlink instance
+ * stream parameters continuously.
+ *
+ * @value 0 Hash check enabled.
```

Comment:
`@boolean`

Reply:
Actually, that probably only makes sense if you want to invert the logic to be parameters that enable.